04-data.Rmd

```diff
@@ -77,6 +77,82 @@ View(pheno_data)
 print(pheno_data[1:5,1:6])
 ```
 
+The GEO record provides information about each of the samples (aka "metadata").  You can use the `table()` function to tabulate how many samples there are for each of the three disease states (along with their corresponding wild type controls).
+
+```{r}
+table( pheno_data$`disease state:ch1` )
+```
+
+::: {.fyi}
+QUESTIONS:
+
+1. Which column contains information about the ages?
+2. Which age has the most samples?
+:::
+
+## Pull in counts
+
+Retrieving GSE162176 using `getGEO()` currently does not obtain RNA-seq expression data.  We have cached a `SummarizedExperiment` object in the [GDSCN datasets](https://anvil.terra.bio/#workspaces/gdscn-exercises/datasets) Workspace.  This object contains metadata, counts, and abundance information as produced by the [nf-co.re/rnaseq/3.6](https://nf-co.re/rnaseq/3.6) pipeline using the [GENCODE M23](https://www.gencodegenes.org/mouse/release_M23.html) annotation.
+
+```{r, echo=FALSE, fig.alt='Screenshot of the GDSCN datasets Workspace showing the cached GSE162176.rds dataset.'}
+ottrpal::include_slide( "https://docs.google.com/presentation/d/1Xd-ZdWF-di4RnQAgF5a6J9fW7eIZtH29QYR7Req3Krc/edit#slide=id.g12145d7e666_0_0" )
+```
+
+The Bioconductor [AnVIL package](http://bioconductor.org/packages/AnVIL) provides a `gsutil_cp()` function to streamline transfers between AnVIL Workspaces.  The following command transfers the `GSE162176.rds` file from the GDSCN datasets Workspace into your Workspace.
+
```

Comment:
Would be nice to add some explanation of what the url inside `gsutil_cp` means, eg pointing to a bucket.